### PR TITLE
Error404

### PR DESCRIPTION
**Pull request by EphraimAssi**
# Description

[Error 404 screens](https://trello.com/c/qhUPjDj9/121-121-error-404-screens)

I made a 404 error page following the Figma design.
image
image

## Acceptance Criteria

It needs to fit the design for mobile and desktop on the Figma design.

## Test Instructions

1. Test a 404 error
2. 404 page should adapt to different screen sizes
3. There should be an English and French report a problem dropdown 

## Help Requested

For medium screen sizes, I didn't really know how to adapt the screen. So any recommendations is welcomed!

## Checklist
Only the report a problem dropdowns have no hard coded strings, I didn't think we needed it for the rest of the page since it's English and French but any feedback welcomed!

- [ ] Strings use placeholders for translation (No hard coded strings)
- [ ] Unit tests have been added/updated

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)

### CODE REVIEW

File: pages/404.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      <section className="layout-container mb-20">
+        <img
+          className="h-auto w-60 pt-6 xl:w-96 xxl:w-1/2"
+          src={"/sig-blk-en.svg"}
+          alt={"Symbol of the Government of Canada"}
+        />
+        <div className="flex flex-col lg:flex-row justify-between items-center lg:items-start mt-8">
+          <div>
+            <div className="relative h-auto xl:w-96 xxl:w-400px lg:w-72 xl:h-400px lg:h-500px mb-8 lg:mb-0">
+              <p className="text-h1 font-bold font-display mb-4">
```
Comment (Moro-Code) [inline pages/404.js:22]: this should probably be an h1 element then

File: pages/404.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+          <div className="flex items-center justify-center circle-background my-8 lg:mt-0">
+            <img
+              className="w-68px lg:w-12 xl:100px"
+              src="/crackedbulb.svg"
+              alt="Cracked lightbulb"
+            />
+          </div>
+          <div>
+            <div className="relative h-auto xl:w-96 xxl:w-400px lg:w-72 xl:h-400px lg:h-500px mb-8 lg:mb-0">
+              <p className="text-h1 font-bold font-display mb-4">
```
Comment (Moro-Code) [inline pages/404.js:55]: same here... should be h1

File: pages/404.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        </div>
+      </section>
+      <footer className="h-100px bg-footer-background-color">
+        <div className="layout-container flex justify-between lg:flex-row-reverse pt-4 lg:pt-0 lg:mt-8">
+          <a
+            href="#"
+            className="block lg:hidden hover:cursor-pointer w-32 text-sm font-body"
+          >
+            <span>Top of page / Haut de la page</span>
+            <img
```
Comment (Moro-Code) [inline pages/404.js:88]: hmmmmm we could we use the icon component for this ? perhaps in a button

**Review by Moro-Code — CHANGES_REQUESTED** [review]
Hey, @EphraimAssi good stuff! A couple of things needed but in general really good! 

The footer element should always be at the bottom of the screen.
Screen Shot 2021-05-31 at 3 11 54 PM

**Comment by EphraimAssi** [comment]
I adjusted the Report a problem in for large screen sizes because the French text wasn't fitting.

File: components/atoms/ActionButton.js [hunk 4]
```diff
@@ -76,11 +81,15 @@ export function ActionButton(props) {
       data-cy-button={props.dataCyButton}
       disabled={props.disabled}
     >
-      {props.icon ? (
+      {props.icon &&
+      (props.iconEnd === undefined || props.iconEnd === false) ? (
```
Comment (Moro-Code) [inline components/atoms/ActionButton.js:85]: you could just use `!props.iconEnd` here as undefined and false are falsey

**Review by Moro-Code — COMMENTED** [review]
Hey looking good to merge. Just one more tiny thing.

**Review by Moro-Code — APPROVED** [review]
@EphraimAssi great great job !! Ship it 🚀

**Pull request merged**